packages/framework/src/Model/Stock/StockSettingsDataFacade.php

```diff
@@ -35,11 +40,7 @@ public function edit(StockSettingsData $stockSettingsData, DomainConfig $domainC
             $domainId,
         );
 
-        $this->setting->setForDomain(
-            Setting::LUIGIS_BOX_RANK,
-            $stockSettingsData->luigisBoxRank,
-            $domainId,
-        );
+        $this->pluginCrudExtensionFacade->saveAllData('luigisBox', static::LUIGIS_BOX_PLUGIN_COMMON_ID, $stockSettingsData->luigisBoxData);
```

Comment:
shouldn't the framework class be independent of the specific usage in individual bundles?
in another words – I don't expect the word luigisbox in framework package

Reply:
yeah, this is our design, see eg. https://github.com/shopsys/shopsys/blob/16.0/packages/framework/src/Model/Product/ProductFacade.php#L178

Reply:
TODO fixit